includes/wikia/api/DesignSystemApiController.class.php

```diff
@@ -165,4 +166,15 @@ private function addCachingHeaders() {
 			$this->response->setCacheValidity( WikiaResponse::CACHE_VERY_SHORT );
 		}
 	}
+
+	private function isWikiaOrgCommunity() {
```

Comment:
As you said, it's not needed anymore

Reply:
This one is actually needed, the one in the model I'll remove :)